projects/scion/microfrontend-platform/src/lib/host/manifest-registry/ɵmanifest-registry.ts

```diff
@@ -137,6 +145,9 @@ export class ɵManifestRegistry implements ManifestRegistry, PreDestroy {
     if (!intention) {
       throw Error(`[IntentionRegisterError] Missing required intention.`);
     }
+    if (intention.qualifier && Object.values(intention.qualifier).some(value => value === '?')) {
```

Comment:
Same here: Should we remove the check for the optional wildcard in a future release? Then, I suggest creating an issue and putting a TODO here.

Reply:
done